BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx

```diff
@@ -27,7 +28,10 @@ class CancelBuildButton extends Component {
     this.setState({
       cancelling: true,
       showModal: false
-    });    
+    });
+
+    const parentPath = window.location.pathname.slice(0, window.location.pathname.lastIndexOf('/'));
```

Comment:
Just pass the `this.props.params` prop down to the build page (starting from the container) and you can build the route from that.